merchantapi/src/main/java/fi/hel/verkkokauppa/configuration/repository/NamespaceRepository.java

```diff
@@ -0,0 +1,16 @@
+package fi.hel.verkkokauppa.configuration.repository;
+
+
+import fi.hel.verkkokauppa.configuration.model.namespace.NamespaceModel;
+import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface NamespaceRepository extends ElasticsearchRepository<NamespaceModel, String> {
+    Optional<NamespaceModel> findByNamespace(String namespace);
+
+    List<NamespaceModel> findAllByNamespace(String namespace);
```

Comment:
Is there a reason it's even possible to have multiple entities with same namespace?

Reply:
There can only be one namespace entity but there can be many merchants under same namespace.

Relation is one to many between namespace and merchants. Using same namespace for merchants creates possibility to create endpoint to change values for merchants in bulk